### PR TITLE
Remove Containers from the standard modules

### PR DESCRIPTION
**Pull request by Kyle-B**
The Vector class supplied by Containers has been subsumed by vector style functions on DefaultRectangular.

### CODE REVIEW

File: test/modules/standard/Vector/capacities.chpl [hunk 1]
```diff
@@ -1,35 +0,0 @@
-use Containers;
```
Comment (thomasvandoren) [inline test/modules/standard/Vector/capacities.chpl]: Do these tests have equivalents for the vector functionality on DefaultRectangular?

**Comment by thomasvandoren** [comment]
Looks good! If the vector functionality on DefaultRectangular doesn't have some of the tests that are removed here, it might be worthwhile to convert them.

**Comment by Kyle-B** [comment]
@daviditen what do you think about converting the tests vs removing them?

**Comment by daviditen** [comment]
Removing them is fine with me.  The DefaultRectangular vector functionality doesn't currently have a notion of capacity or offset outside of the underlying domain value.  It doesn't currently do the doubling and halving in size that seems to be the main focus of these tests.

**Pull request merged**